### PR TITLE
SAK-50454 Polls improve voting ui responsiveness

### PR DESCRIPTION
**Pull request by Sharadhi98**
I have resolved the responsiveness issue of the polls tool by using table-responsive. I observed that the grid system applied to the portal-main-container was interfering with the styling of table-responsive, so I replaced it with flex and added two classes: d-flex and flex-column. 
I'd love some feedback on changing the portal-main-container from grid to flex. 
Thanks!

### CODE REVIEW

File: polls/tool/src/webapp/templates/votePolls.html [hunk 1]
```diff
@@ -108,9 +109,9 @@ <h1 rsf:id="poll-list-title">Polls</h1>
       </td>
     </tr>
   </tbody>
-</table>
+</table></div>
 
-<p class="act">
+<p class="act" style="flex-wrap:nowrap;">
```
Comment (ern) [inline polls/tool/src/webapp/templates/votePolls.html:114]: prefer not using inline style

Comment (Sharadhi98) [reply polls/tool/src/webapp/templates/votePolls.html]: Thanks! I updated the code with a bootstrap class instead.

File: polls/tool/src/webapp/templates/votePolls.html [hunk 2]
```diff
@@ -108,9 +109,9 @@ <h1 rsf:id="poll-list-title">Polls</h1>
       </td>
     </tr>
   </tbody>
-</table>
+</table></div>
```
Comment (kunaljaykam) [inline polls/tool/src/webapp/templates/votePolls.html:112]: Nice work! Just a quick suggestion: placing </table> and </div> on separate lines improves readability and maintainability. Could you adjust the formatting?

**Review by kunaljaykam — CHANGES_REQUESTED** [review]

**Review by adrianfish — CHANGES_REQUESTED** [review]
I've added another commit. Don't be afraid to remove redundant code, like the javascript script type on script tags. If I'm working in a file and I see old stuff that might not actually be related to the thing I'm fixing, I will still clean it up. That way our codebase steadily improves. Don't go crazy and start looking at the entire polls tool, but changes to the area where you're working are fine.

File: polls/tool/src/webapp/templates/votePolls.html [hunk 3]
```diff
@@ -1,120 +1,122 @@
...
-
-<p class="act">
-<input type="submit" rsf:id="delete-polls" id="delete-polls" name="deleteSelectPoll" value="Update"  accesskey="s" disabled="disabled" />
-<input type="submit" rsf:id="reset-polls-votes" id="reset-polls-votes" name="resetSelectPoll" value="Reset"  accesskey="r" disabled="disabled" />
-</p>
-</form>
-</div>
-</body>
-</html>
+<!DOCTYPE html      PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
```
Comment (kunaljaykam) [inline polls/tool/src/webapp/templates/votePolls.html:1]:
```suggestion
<!DOCTYPE html>
```

File: polls/tool/src/webapp/templates/votePolls.html [hunk 4]
```diff
@@ -1,120 +1,122 @@
...
+<html rsf:id="polls-html" xmlns="http://www.w3.org/1999/xhtml" xmlns:rsf="http://ponder.org.uk/rsf">
+  <head >
+    <title>Polls</title>
+    <link rsf:id="scr=portal-matter" href="https://vula.uct.ac.za/library/skin/tool_base.css" type="text/css" rel="stylesheet" media="all"/>
+    <link href="https://vula.uct.ac.za/library/skin/default-skin/tool.css" type="text/css" rel="stylesheet" media="all" rsf:id="scr=portal-matter" />
+    <script  rsf:id="scr=portal-matter" src="/library/js/headscripts.js"></script>
+    <script>includeLatestJQuery('polls');</script>
+    <script>includeWebjarLibrary('jquery.tablesorter');</script>
```
Comment (kunaljaykam) [inline polls/tool/src/webapp/templates/votePolls.html:9]:
```suggestion
<script>
    includeLatestJQuery('polls');
    includeWebjarLibrary('jquery.tablesorter');
</script>
```

**Review by kunaljaykam — APPROVED** [review]
Looks good overall! However, I noticed there's no i18n support, @adrianfish  Is it okay? @sharadhi, please test one last time with the latest changes, and once everything looks good, we can merge. Thanks!

**Comment by Sharadhi98** [comment]
> Looks good overall! However, I noticed there's no i18n support, @adrianfish Is it okay?
> 
> @Sharadhi98 , please test one last time with the latest changes, and once everything looks good, we can merge. Thanks!

For some reason, I'm unable to test the Polls tool. Even though there's are no compilation errors, I get an error when I load the tool on my local. Here's a screenshot.
Screen Shot 2024-09-23 at 6 15 56 PM

**Comment by adrianfish** [comment]
@Sharadhi98 I'll pull your branch again and test things out. If it's all good, I'll approve it and merge.

**Review by adrianfish — APPROVED** [review]
Builds and works fine. Thanks Sharadhi.

**Pull request merged**